st3/lsp_utils/_client_handler/abstract_plugin.py

```diff
@@ -128,12 +130,16 @@ def can_start(cls, window: sublime.Window, initiating_view: sublime.View,
     @classmethod
     def on_pre_start(cls, window: sublime.Window, initiating_view: sublime.View,
                      workspace_folders: List[WorkspaceFolder], configuration: ClientConfig) -> Optional[str]:
-        extra_paths = path.pathsep.join(cls.get_additional_paths())
+        # To fix https://github.com/TerminalFi/LSP-copilot/issues/163 ,
+        # We don't want to add the same path multiple times whenever a new server session is created.
```

Comment:
This comment could be placed in more relevant place. Now it looks like it applies to the whole method.

Reply:
Now it should be relevant I guess.